### ISSUE TITLE
resetDB too slow for testing / Need more lightweight implementation

### ISSUE THREAD

**Issue by arkocal** [Open-IoT-Service-Platform/oisp-frontend#2]
For client side testing, database needs to be resetted often to keep test isolated. The current implementation in `admin/resetDB.js` drops all tables, which is practical when tables are large, but for smaller testcases it could be faster to delete entries instead. Currenty I need 10+ seconds for each reset, which adds up quite fast.

Below is the tear down function in the python unittests I use. When the function is called, there are no more than 3 entries in any table.

```python
    def tearDown(self):
        started_at = time.time()
        print("Resetting DB")
        os.system("docker exec -it platformlauncher_dashboard_1 node "
                  "/app/admin resetDB &> /dev/null")
        os.system("docker exec -it platformlauncher_dashboard_1 node "
                  "/app/admin addUser {} {} {} "
                  "&> /dev/null".format(config.username,
                                        config.password,
                                        config.role))
        print("Reset database in {} seconds".format(time.time()-started_at))
```
(outputs about 12 seconds every time)

**Comment by nevets963** [comment]
@arkocal This is true and simply truncating all tables is the desired implementation. I tried to go with this approach but then I run foreign key constraints being violated and then after looking for a while how to temporarily disable this for Postgres 9.4, it was easier to simply drop all tables and re-add them. As this is mainly for the end-to-end test suite, I wouldn't imagine it's that bad as people will mostly work on the individual repo's (with separate test suites) and then have Jenkins run the e2e test suite to ensure the whole system continues to function.

**Comment by arkocal** [comment]
```SQL
psql -U postgres -d iot -t -c "SELECT table_name FROM
information_schema.tables WHERE table_type='BASE TABLE' AND
table_schema='dashboard' AND table_name <> 'user_accounts';
```

```SQL
psql -U postgres -d iot -t -c 'TRUNCATE {RESULT FROM FIRST QUERY} CASCADE;' """
```

Running those two commands directly on the postgres container worked for me (less than a second/reset) . It does delete the default users though.
I would not recommend implementing this exactly this way, but I am leaving it as a comment, as others might need it.
